<launch>
  <node 
    pkg="livox_camera_calib"
    type="lidar_camera_calib"
    name="lidar_camera_calib"
    args="$(find livox_camera_calib)/config/camera_0231.yaml $(find livox_camera_calib)/config/config_indoor_robosense-m1_ar0231.yaml
   /mnt/data/ros/livox_camera_calib/robosense-m1_ar0231/2021-12-03-01-22-33.bag $(find livox_camera_calib)/result"
    output="screen"
  />
  <node 
    pkg="rviz"
    type="rviz"
    name="rviz"
    args="-d $(find livox_camera_calib)/rviz_cfg/calib_indoor.rviz"
  />
</launch>
